### PR TITLE
adds transcripts to ru301 pages w/ video

### PR DESCRIPTION
**Pull request by justincastilla**
Adds transcripts to all RU301 pages for SEO. Also increased size of videos as transcripts no longer appear next to video embed.

### CODE REVIEW

File: docs/operate/redis-at-scale/high-availability/basic-replication/index-basic-replication.mdx [hunk 1]
```diff
@@ -8,4 +8,24 @@ isEditable: false
...
\ No newline at end of file
+<div align="center">
+<iframe width="896" height="509" src="https://www.youtube.com/embed/-osCdf90tRA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+<br />
+<br />
+<p>Replication in Redis follows a simple primary-replica model where the replication happens in one direction - from the primary to one or multiple replicas. Data is only written to the primary instance and replicas are kept in sync so that they’re exact copies of the primaries.</p>
+
+To create a replica, you instantiate a Redis server instance with the configuration directive <strong>replicaof</strong> set to the address and port of the primary instance. once the replica instance is up and running, the replica will try to sync with the primary. To transfer all of its data as efficiently as possible, the primary instance will produce a compacted version of the data in a snapshot (.rdb) file and send it to the replica.
```
Comment (guyroyse) [inline docs/operate/redis-at-scale/high-availability/basic-replication/index-basic-replication.mdx]: Capital O in once.

File: docs/operate/redis-at-scale/index-redis-at-scale.mdx [hunk 2]
```diff
@@ -11,9 +11,31 @@ import RedisCard from '@site/src/theme/RedisCard';
...
+
+There are a bunch of reasons for this. For example, maybe you're a large enterprise with your own data centers and dedicated ops teams. Or perhaps you're a mission-critical application whose SLAs are so rigid that you need to be able to dig deeply into any potential performance issue. This often rules out cloud-based deployments, since the cloud hides away the hardware and networks you're operating in. In this case, you're deploying Redis on your own. And for that, you need to know how Redis scales.
+
+Learning this isn't just useful; it's also genuinely interesting. Sharding, replication, high availability, and disaster recovery are all important concepts that anyone can understand with the right explanation. These concepts aren't rocket science. They're no harder to understand than basic high school math, and knowing about them makes you a better developer.
+In this course, we'll look closely at how open source Redis scales. And you'll learn by doing, as we present a lot of the ideas through hands-on labs.
```
Comment (guyroyse) [inline docs/operate/redis-at-scale/index-redis-at-scale.mdx]: Do we need a new-line here?

File: docs/operate/redis-at-scale/persistence-and-durability/persistence-options-in-redis/index-persistence-options-in-redis.mdx [hunk 3]
```diff
@@ -8,4 +8,29 @@ isEditable: false
...
+
+Creating a .rdb file requires a lot of disk I/O. If performed in the main Redis process, this would reduce the server’s performance. That’s why this work is done by a forked child process. But even forking can be time-consuming if the dataset is large. This may result in decreased performance or in Redis failing to serve clients for a few milliseconds or even up to a second for very large datasets. Understanding this should help you decide whether this solution makes sense for your requirements.
+
+You can configure the name and location of the .rdb file with the <strong>dbfilename</strong> and <strong>dir</strong> configuration directives, either through the <strong>redis.conf</strong> file, or through the redis-cli as explained in [Section 1 Unit 2](http://localhost:3000/operate/redis-at-scale/talking-to-redis/configuring-a-redis-server). And of course you can configure how often you want to create a snapshot. Here’s an excerpt from the redis.conf file showing the default values.
```
Comment (guyroyse) [inline docs/operate/redis-at-scale/persistence-and-durability/persistence-options-in-redis/index-persistence-options-in-redis.mdx]: I don't know how all the coursework stuff works, but is a localhost URL valid?

File: docs/operate/redis-at-scale/scalability/clustering-in-redis/index-scalability.mdx [hunk 4]
```diff
@@ -9,4 +9,50 @@ isEditable: false
...
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<iframe width="586" height="330" src="https://www.youtube.com/embed/jJMJc9QZaoA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
\ No newline at end of file
+<div align="center">
+<iframe width="896" height="509" src="https://www.youtube.com/embed/jJMJc9QZaoA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+<br />
+<br />
+<p>Before we jump into the details, let's first address the elephant in the room: DBaaS offerings, or "database-as-a-service" in the cloud. No doubt, it's useful to know how Redis scales and how you might deploy it. But deploying and maintaining a Redis cluster is a fair amount of work. So if you don't want to deploy and manage Redis yourself, then consider signing up for a Redis Cloud, our managed service, and let us do the scaling for you. Of course, that route is not for everyone. And as I said, there's a lot to learn here, so let's dive in.</p>
```
Comment (guyroyse) [inline docs/operate/redis-at-scale/scalability/clustering-in-redis/index-scalability.mdx]: Redis Cloud, not a Redis Cloud

File: docs/operate/redis-at-scale/talking-to-redis/redis-clients/index-redis-clients.mdx [hunk 5]
```diff
@@ -8,4 +8,21 @@ isEditable: false
...
+    <iframe width="896" height="509" src="https://www.youtube.com/embed/2LRXM6ELhFw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+<br />
+<br />
+<p>Redis has a client-server architecture and uses a request-response model. Applications send requests to the Redis server, which processes them and returns responses for each. The role of a Redis client library is to act as an intermediary between your application and the Redis server.</p>
+
+Client libraries perform the following duties:
+- Implement the Redis wire protocol - the format used to send requests to and receive responses from the Redis server
+- Provide an idiomatic API for using Redis commands from a particular programming language
+Managing the connection to Redis
```
Comment (guyroyse) [inline docs/operate/redis-at-scale/talking-to-redis/redis-clients/index-redis-clients.mdx]: This looks malformed.

File: docs/operate/redis-at-scale/talking-to-redis/redis-server-overview/index-redis-server-overview.mdx [hunk 6]
```diff
@@ -8,4 +8,21 @@ isEditable: false
...
+<div align="center">
+<iframe width="896" height="504" src="https://www.youtube.com/embed/GnVvrLF7oeE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+<br />
+<br />
+<p>As you might already know, Redis is an open source data structure server written in C. You can store multiple data types, like strings, hashes, and streams and access them by a unique key name. </p>
+
+For example, if you have a string value “Hello World” saved under the key name “greeting”, you can access it by running the GET command followed by the key name - greeting. All keys in a Redis database are stored in a flat keyspace. There is no enforced schema or naming policy, and the responsibility for organizing the keyspace is left to the developer.
+
+The speed Redis is famous for is mostly due to the fact that Redis stores and serves data entirely from RAM memory instead of disk, as most other databases do. Another contributing factor is its predominantly single-threaded nature: single-threading avoids race conditions and CPU-heavy context switching associated with threads.
```
Comment (guyroyse) [inline docs/operate/redis-at-scale/talking-to-redis/redis-server-overview/index-redis-server-overview.mdx]: RAM memory is a TLA acronym. ;)

File: docs/operate/redis-at-scale/talking-to-redis/redis-server-overview/index-redis-server-overview.mdx [hunk 7]
```diff
@@ -8,4 +8,21 @@ isEditable: false
...
+
+Let’s now look at exactly what happens behind the scenes with every Redis request. When a client sends a request to a Redis server, the request is first read from the socket, then parsed and processed and finally, the response is written back to the socket and sent to the user. The reading and especially writing to a socket are expensive operations, so in Redis version 6.0 multi-threaded I/O was introduced. When this feature is enabled, Redis can delegate the time spent reading and writing to I/O sockets over to other threads, freeing up cycles for storing and retrieving data and boosting overall performance by up to a factor of two for some workloads.
+
+Throughout the rest of the section, you’ll learn how to use the Redis command line interface, how to configure your Redis server, and how to choose and tune your Redis client library.
```
Comment (guyroyse) [inline docs/operate/redis-at-scale/talking-to-redis/redis-server-overview/index-redis-server-overview.mdx]: Is command-line hyphenated?

**Review by guyroyse — CHANGES_REQUESTED** [review]
Just minor typos and stuff.

**Comment by simonprickett** [comment]
I took a look too and think this will be good to go with @guyroyse 's suggestions.  Thanks everyone.

**Pull request merged**